experimental/traffic-portal/src/app/api/delivery-service.service.ts

```diff
@@ -441,4 +442,16 @@ export class DeliveryServiceService extends APIService {
 		this.deliveryServiceTypes = r;
 		return r;
 	}
+
+	/**
+	 * Gets a Delivery Service's SSL Keys
+	 *
+	 * @param ds The delivery service xmlid or object
+	 * @returns The DS ssl keys
+	 */
+	public async getSSLKeys(ds: string | ResponseDeliveryService): Promise<ResponseDeliveryServiceSSLKey> {
+		const xmlId = typeof ds === "string" ? ds : ds.xmlId;
```

Comment:
Coverage of branches in this file is reduced by this PR from 100% to 97.36% - that's because the path where a string is passed to this method is not covered.